autopilot/autopilot.go

```diff
@@ -43,7 +43,7 @@ type Bus interface {
 
 	// hostdb
 	Host(ctx context.Context, hostKey types.PublicKey) (hostdb.HostInfo, error)
-	Hosts(ctx context.Context, offset, limit int) ([]hostdb.Host, error)
+	Hosts(ctx context.Context, offset, limit int, includeBlocked bool, addressContains string, keyIn []types.PublicKey) ([]hostdb.Host, error)
```

Comment:
I'm not really a big fan of `addressContains` and `keyIn`. We would usually do `keyIn` by sending a POST request listing all the keys in the body (seeing as query string params are capped). I think `addressContains` is the perfect reason to add fuzzy host search...

`includeBlocked` is perfectly fine 👍 was wondering when we were going to have to add that one